doc/source/_templates/sidebar-extras.html

```diff
@@ -0,0 +1,14 @@
+<div class="bd-toc-item navbar-nav">
+    <p aria-level="2" class="caption" role="heading">
+        <span class="caption-text"><b>{{ _('Extras') }}</b></span>
+    </p>
+    <ul class="nav bd-sidenav">
+    <li class="toctree-l1"><a class="reference internal" href="../glossary.html">Glossary</a></li>
+
+    <li class="toctree-l1"><a class="reference internal" href="../release.html">Release notes</a></li>
+
+    <li class="toctree-l1"><a class="reference internal" href="../license.html">NumPy license</a></li>
+
+    <li class="toctree-l1"><a class="reference internal" href="https: //numpy.org/neps">NumPy Enhancement Proposals (NEPs)</a></li>
```

Comment:
Is the space between `:` and `//` intended? It seems to work, but is somewhat unusual. None of the links work in the circleci generated document, but that is probably unrelated.